src/sentry/static/sentry/app/views/settings/components/emptyMessage.tsx

```diff
@@ -49,7 +50,25 @@ const Action = styled('div')`
   ${MarginStyles};
 `;
 
-const EmptyMessage = ({title, description, icon, children, action, size, className}) => (
+type Props = {
+  title?: React.ReactNode;
+  description?: React.ReactNode;
+  icon?: string;
+  action?: React.ReactElement;
+  size?: 'large' | 'medium';
```

Comment:
@nolachen What does `size=medium` even do? Seems like it's a meaningless option.

Reply:
@scefali  yeah only `size=large` would actually change the size. Not sure if we want to change how it's implemented now, maybe this is so that there could potentially be other sizes in the future?

Reply:
@nolachen Yea, let's just add a comment about it for now and leave it alone.